README.md

```diff
@@ -239,24 +239,290 @@ One can also do it manually:
 
       $ dotnet new -u Equinox.Templates
 
+<a name="guidance"></a>
 # PATTERNS / GUIDANCE
 
-## Use Strongly typed ids
+<a name="tldr"></a>
+## TL;DR
 
-Wherever possible, the samples strongly type identifiers, particularly ones that might naturally be represented as primitives, i.e. `string` etc.
+1. DO have global strongly typed id types in `namespace Domain`
+2. DONT have global `module Types`. AVOID per Aggregate `module Types` or top level `type` definitions
+3. DO group stuff predictably per `module Aggregate`: `Stream, Events, Reactions, Fold, Decisions, Service, Factory`. And keep grouping within that.
+4. DONT `open <Aggregate>`, `open <Aggregate>.Events`, `open <Aggregate>.Fold`
+5. DO design for idempotency everywhere
+6. DONT use `Result` or a per-Aggregate `type Error`. DO use minimal result types per decision function
+7. DONT pass out `Fold.State` from a `Service`
+8. DONT be a slave to CQRS for all read paths. CONSIDER AllowState. AVOID Query
+9. DONT be a slave to the Command pattern
```

Comment:
We might as well go all out and discourage the pattern entirely. The way Equinox code is written the command pattern, even in its simplest state, results in more harder to read code.

```fsharp
module Decide =
    type Command = Increment | Decrement
    let decide command state =
        match command with
        | Increment -> if state > 10 then [] else [Events.Incremented]
        | Decrement -> if state = 0 then [] else [Events.Decremented]
```

vs

```fsharp
module Decide =
    let increment state = if state > 10 then [] else [Events.Incremented]
    let decrement state = if state = 0 then [] else [Events.Decremented]
```

Reply:
yes, or `let increment state = if state > 10 then [] else [Events.Incremented]` => `let increment state = [if state <= 10 then Events.Incremented]`

Reply:
attempted to cover this in [283727e](https://github.com/jet/dotnet-templates/pull/133/commits/283727e82cc717f5e62093d0c373ab730edc9979) but not really satisfied as I went too far. I hate the Counter example!